addons/im_livechat/static/src/core/web/channel_invitation_patch.xml

```diff
@@ -2,14 +2,14 @@
 <templates xml:space="preserve">
     <t t-inherit="discuss.ChannelInvitation-selectableItem" t-inherit-mode="extension">
         <xpath expr="//*[@name='selectablePartnerName']" position="inside">
-            <span t-if="props.thread?.channel_type === 'livechat' and selectablePartner.user_livechat_username" t-esc="selectablePartner.user_livechat_username" class="text-truncate text-muted smaller mx-2"/>
-            <span t-if="props.thread?.channel_type === 'livechat' and selectablePartner.is_in_call" class="flex-shrink-0 opacity-75 smaller mx-2">
+            <span t-if="props.thread?.channel?.channel_type === 'livechat' and selectablePartner.user_livechat_username" t-esc="selectablePartner.user_livechat_username" class="text-truncate text-muted smaller mx-2"/>
```

Comment:
Ok for another PR as the diff is surprisingly rather small here, but channel invitation should use channel.